src/views/Checkout.vue

```diff
@@ -154,6 +167,9 @@ class Checkout extends Vue {
         const minWidth = 675; // Width below which disclaimer would break into three lines.
         const minHeight = 890; // Height at which two lines fit at bottom, also if logos over carousel shown.
         this.screenFitsDisclaimer = window.innerWidth >= minWidth && window.innerHeight >= minHeight;
+        // Throttle calls to carousel.updateDimensions as its an expensive call
+        clearTimeout(this.dimensionsUpdateDelay);
+        this.dimensionsUpdateDelay = window.setTimeout(() => (this.$refs.carousel as Carousel).updateDimensions(), 150);
```

Comment:
As a "delay" is the time (150ms in this case), a better name for this variable would be either `dimensionsUpdateThrottle` or `dimensionsUpdateTimeout`, I think.
```suggestion
        this.dimensionsUpdateThrottle = window.setTimeout(() => (this.$refs.carousel as Carousel).updateDimensions(), 150);
```